examples/lighting-app/bouffalolab/bl702/BUILD.gn

```diff
@@ -149,6 +149,10 @@ bouffalolab_executable("lighting_app") {
     defines += [ "BOOT_PIN_RESET=31" ]
   }
 
+  if (chip_enable_factory_data) {
+    defines += [ "CONFIG_BOUFFALOLAB_FACTORY_DATA_ENABLE" ]
```

Comment:
CONFIG_BOUFFALOLAB_FACTORY_DATA_ENABLE=1 for consistency with the other defines.

Reply:
Same comment below.

Reply:
changed